### PR TITLE
New Presence: Apple

### PR DESCRIPTION
**Pull request by drull1000**
A presence for Apple's Website
apple bag
apple browsering
apple buying
apple checking
apple favorites
apple initial
apple iOS
apple news
apple service
apple shop

### CODE REVIEW

**Comment by Bas950** [comment]
maybe make it so it has a transparent background?

**Comment by drull1000** [comment]
> maybe make it so it has a transparent background?
Choose one
apple preto
apple white

**Comment by Bas950** [comment]
second one

**Comment by drull1000** [comment]
> second one
> (white)

Done

**Comment by drull1000** [comment]
> > second one
> > (white)
> 
> Done

I changed the image on the discord website, so I won't need to make any changes in the code

File: Apple/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      "id": "244953147154366464"
+    },
+    "url": "www.apple.com",
+    "description": {
+      "pt_br": "Apple Inc. é uma empresa multinacional norte-americana que tem o objetivo de projetar e comercializar produtos eletrônicos de consumo, software de computador e computadores pessoais.",
+      "en": " Apple Inc. is an American multinational company that aims to design and market consumer electronics, computer software and personal computers."
+    },
+    "service": "Apple",
+    "version": "1.0.0",
+    "logo": "https://i.imgur.com/4d0J3aC.png",
```
Comment (frozaum) [inline Apple/dist/metadata.json:13]: nothing necessary, but can you change the logo?

File: Apple/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    "thumbnail": "https://i.imgur.com/D5b7fQ5.jpg",
+    "color": "#606C80",
+    "tags": [
+      "apple ",
```
Comment (frozaum) [inline Apple/dist/metadata.json:17]: remove the backspace

Comment (frozaum) [inline Apple/dist/metadata.json:17]: remove the backspace

**Pull request merged**